airflow/contrib/hooks/spark_submit_hook.py

```diff
@@ -468,3 +491,24 @@ def on_kill(self):
                                              stderr=subprocess.PIPE)
 
                 self.log.info("YARN killed with return code: %s", yarn_kill.wait())
+
+            if self._kubernetes_driver_pod:
+                self.log.info('Killing pod {} on Kubernetes'
+                              .format(self._kubernetes_driver_pod))
+
+                # For now, only instantiate Kubernetes client when killing a spark pod.
+                # Use Default namespace only for now. Can be retrieved from the
+                # spark.kubernetes.namespace submit option.
+                try:
+                    client = kube_client.get_kube_client()
+                    api_response = client.delete_namespaced_pod(
+                        self._kubernetes_driver_pod,
+                        "default",
```

Comment:
Please make the namespace configurable using the connection

Reply:
Do you think it is better to add a new namespace param to the operator, or check to see if spark.kubernetes.namespace has been set in the existing --conf parameter?